source/src/main/java/com/cyb/tms/controller/TmsModuleController.java

```diff
@@ -0,0 +1,55 @@
+package com.cyb.tms.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import com.cyb.tms.dto.TmsModuleDTO;
+import com.cyb.tms.entity.TmsModule;
+import com.cyb.tms.entity.TmsProject;
+import com.cyb.tms.entity.TmsUsers;
+import com.cyb.tms.service.TmsModuleService;
+import com.cyb.tms.service.TmsProjectService;
+import com.cyb.tms.util.URIConstants;
+
+public class TmsModuleController {

```

Comment:
@anuph9 @RestController should be define at the top of TmsModuleController class